Ci test - IGNORE - DO NOT MERGE

heinsaar opened the pull request:
#### Testing

Testing

heinsaar committed:
`CI` Test run - should fail - wrong discriminator

heinsaar committed:
CI test

heinsaar committed:
Test

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=39064) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=39064) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/project-chip/connectedhomeip?pullRequest=39064) it.</sub>

vivien-apple committed:
[Darwin] Convert PlatformManagerImpl.cpp to .mm (#39059)

* [Darwin] Convert PlatformManagerImpl.cpp to .mm

* [Darwin] clang-format src/platform/Darwin/PlatformManagerImpl.mm

s07641069 committed:
Update compatible builds to docker version 132 (#39030)

bhmanda-silabs committed:
[Silabs] [WiFi] Add lighting RGB color feature for 917 SoC BRD2605A devkit board (#37678)

* Added RGB support for 917 SoC BRD2605A devkit board

* Removed logs which are added for validation

* Resolving review comments

* Resolved review comments on colorFormat.cpp

* Resolved review comments

* Updated review comments

* Clean up and optimize the code for readabilty

* Removed commented code

* Added sLightLED object based on LED type the board supports

* Resolved review comments

* Resolved build errors

* Removed ColorFormat src and heder into common folder and modified gn files

* Resolved build error for 917 SoC devkit boards

* Added RGBLED changes into separate folder and changes for level action

* Resolved build error by adding app namespace

* Added rgb led state function in RGBLEDWidget

* Updated the PostAttribute callback to read and pass it to the lighting manager

* Resolved review comments

* Resolved build error with 917 SoC BRD4338A

* Update examples/lighting-app/silabs/src/LightingManager.cpp

Co-authored-by: Junior Martinez <67972863+jmartinez-silabs@users.noreply.github.com>

* Update examples/lighting-app/silabs/src/LightingManager.cpp

Co-authored-by: Junior Martinez <67972863+jmartinez-silabs@users.noreply.github.com>

* Resolved review and cleanup

* Added GetLevel function for mLevel

* Update examples/lighting-app/silabs/src/LightingManager.cpp

Co-authored-by: Junior Martinez <67972863+jmartinez-silabs@users.noreply.github.com>

* Resolved review comments

* Resolved review comments

* Restyled by gn

* Added std:clamp and included Accessors.h to resolve build error

---------

Co-authored-by: Junior Martinez <67972863+jmartinez-silabs@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

j-ororke committed:
[Follow-Up] Updating TC_ACL_2_6 to add subscription validation and fix for issue #38663 (#38705)

* [Follow-Up] Updating TC_ACL_2_6 to add subscription validation for read events
- Added additional subscription validation to test steps 3, 5, and 7
- Follow-Up for follow-up task here: https://github.com/project-chip/matter-test-scripts/issues/548

* Restyled by autopep8

* Restyled by isort

* Resolving linting errros

* Updating TC_ACL_2_6 test module:
- Changing step 7 back to remove the subscription as this step expects no new entries

* Updating TC_ACL_2_6 python3 test module:
- Removing for retry loop in test step 5
- Moving function out of main test block and into a class function

* Restyled by autopep8

* Updating TC_ACL_2_6 python3 test module:
- Resolving linting errors

* Adding value fix for https://github.com/project-chip/connectedhomeip/issues/38663

* Update src/python_testing/TC_ACL_2_6.py

Adding new assert in to code from Cecille.

Co-authored-by: C Freeman <cecille@google.com>

---------

Co-authored-by: Restyled.io <commits@restyled.io>
Co-authored-by: C Freeman <cecille@google.com>

j-ororke committed:
[Fix] Updating import for matter_testing submodule in CADMIN_1_3_4 and CADMIN_1_11 python3 test modules (#38925)

* [Fix] Updating import location for CADMIN_1_3_4 and CADMIN_1_11 test modules:
- Changing import location to resolve issue with running test modules CADMIN_1_3_4 and CADMIN_1_11 in TH UI
- Fixes issue: [567](https://github.com/project-chip/matter-test-scripts/issues/567)

* Restyled by autopep8

* Restyled by isort

---------

Co-authored-by: Restyled.io <commits@restyled.io>

sxb427 committed:
Add folder specific review rules (#39037)

andy31415 committed:
Move NXP zephyr build to docker image 129 (#39078)

Co-authored-by: Andrei Litvin <andreilitvin@google.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=39064) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=39064) before we can accept your contribution.<br/>**5** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: s07641069<br/>:white_check_mark: j-ororke<br/>:white_check_mark: bhmanda-silabs<br/>:white_check_mark: andy31415<br/>:white_check_mark: sxb427<br/>:x: heinsaar<br/>:x: vivien-apple<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/project-chip/connectedhomeip?pullRequest=39064) it.</sub>
